sandbox-invoke-mocks.js

```diff
@@ -0,0 +1,10 @@
hFb3R2b3MgVS4pLCBLLiBUb3JpZ29lIChIaXJvc2hpbWEKVS4pLCBOLiBLb2dpc28sIE0uIFlvbmV5YW1hIChPc2FrYSBNZXRyb3BvbGl0YW4gVS4pLCBNLiBNb3JpdGFraSAoVS4KVG9reW8pLCBULiBLYW5vIChVLiBNaWNoaWdhbikgLS0gdGhlIEdSQkFscGhhIGNvbGxhYm9yYXRpb24uCgpUaGUgbG9uZy1kdXJhdGlvbiBHUkIgMjMwMjA3QiAoQUdJTEUvTUNBTCBkZXRlY3Rpb246IEdDTiBDaXJjLiAzMzI5NjsKU3dpZnQvQkFULUdVQU5PIGRldGVjdGlvbjogR0NOIENpcmMuIDMzMjk4OyBDQUxFVC9HQ0JNIGRldGVjdGlvbjogdHJpZ2dlcgpuby4gMTM1OTc3OTk0NTsgSU5URUdSQUwvU1BJLUFDUyBkZXRlY3Rpb246IHRyaWdnZXIgbm8uIDEwMTkwKSB3YXMgb2JzZXJ2ZWQKYnkgdGhlIEdSQkFscGhhIDFVIEN1YmVTYXQgKFBhbCBldCBhbC4gUHJvYy4gU1BJRSAyMDIwKS4KClRoZSBkZXRlY3Rpb24gd2FzIGNvbmZpcm1lZCBhdCB0aGUgcGVhayB0aW1lIDIwMjMtMDItMDcgMDQ6NDA6NDggVVRDLiBUaGUKVDkwIGR1cmF0aW9uIG1lYXN1cmVkIGJ5IEdSQkFscGhhIGlzIDEwIHMgYW5kIHRoZSBvdmVyYWxsIHNpZ25pZmljYW5jZQpkdXJpbmcgVDkwIHJlYWNoZXMgMjUgc2lnbWEuCgpUaGUgbGlnaHQgY3VydmUgb2J0YWluZWQgYnkgR1JCQWxwaGEgaXMgYXZhaWxhYmxlIGhlcmU6CgpodHRwczovL2djYzAyLnNhZmVsaW5rcy5wcm90ZWN0aW9uLm91dGxvb2suY29tLz91cmw9M0RodHRwcyUzQSUyRiUyRmdyYmFscGg9CmEua29ua29seS5odSUyRnN0YXRpYyUyRnNoYXJlJTJGR1JCMjMwMjA3Ql9HQ04ucGRmJmRhdGE9M0QwNSU3QzAxJTdDY2lyY3VsYT0KcnMlNDBkZXYuZ2NuLm5hc2EuZ292JTdDM2ZlMjc3YTYzOGZmNDk4MWYxY2IwOGRiMGI4YmIzMDElN0M3MDA1ZDQ1ODQ1YmU0OGFlPQo4MTQwZDQzZGE5NmRkMTdiJTdDMCU3QzAlN0M2MzgxMTY0Njc1MjgyMzYwMzIlN0NVbmtub3duJTdDVFdGcGJHWnNiM2Q4ZXlKV0k9CmpvaU1DNHdMakF3TURBaUxDSlFJam9pVjJsdU16SWlMQ0pCVGlJNklrMWhhV3dpTENKWFZDSTZNbjAlM0QlN0MzMDAwJTdDJTdDJT0KN0Mmc2RhdGE9M0RlVEVlM1U4NnZCYk9jYVc4YzBid01oT1YwOVB1TG1oTDJ2WUxMazN4N2dZJTNEJnJlc2VydmVkPTNEMAoKQWxsIEdSQkFscGhhIGRldGVjdGlvbnMgYXJlIGxpc3RlZCBhdDoKaHR0cHM6Ly9nY2MwMi5zYWZlbGlua3MucHJvdGVjdGlvbi5vdXRsb29rLmNvbS8/dXJsPTNEaHR0cHMlM0ElMkYlMkZtb25vY2VyPQpvcy5waHlzaWNzLm11bmkuY3olMkZoZWElMkZHUkJBbHBoYSUyRiZkYXRhPTNEMDUlN0MwMSU3Q2NpcmN1bGFycyU0MGRldi5nY249Ci5uYXNhLmdvdiU3QzNmZTI3N2E2MzhmZjQ5ODFmMWNiMDhkYjBiOGJiMzAxJTdDNzAwNWQ0NTg0NWJlNDhhZTgxNDBkNDNkYTk2ZD0KZDE3YiU3QzAlN0MwJTdDNjM4MTE2NDY3NTI4MjM2MDMyJTdDVW5rbm93biU3Q1RXRnBiR1pzYjNkOGV5SldJam9pTUM0d0xqQXdNPQpEQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazFoYVd3aUxDSlhWQ0k2TW4wJTNEJTdDMzAwMCU3QyU3QyU3QyZzZGF0YT0zRHc9CmIlMkZLbG1lQkElMkZzdlQ3UUNwNkNlaGphdW9pZjRVRW9KNCUyRk1sYzhMbjlrMCUzRCZyZXNlcnZlZD0zRDAKCkdSQkFscGhhLCBsYXVuY2hlZCBvbiAyMDIxIE1hcmNoIDIyLCBpcyBhIGRlbW9uc3RyYXRpb24gbWlzc2lvbiBmb3IgYQpmdXR1cmUgQ3ViZVNhdCBjb25zdGVsbGF0aW9uIChXZXJuZXIgZXQgYWwuIFByb2MuIFNQSUUgMjAxOCkuIFRoZSBkZXRlY3RvcgpvZiBHUkJBbHBoYSBjb25zaXN0cyBvZiBhIDc1IHggNzUgeCA1IG1tMyBDc0kgc2NpbnRpbGxhdG9yIHJlYWQgb3V0IGJ5IGEKU2lQTSBhcnJheSwgY292ZXJpbmcgdGhlIGVuZXJneSByYW5nZSBmcm9tIH41MCBrZVYgdG8gfjEwMDAga2VWLiBUbwppbmNyZWFzZSB0aGUgZHV0eSBjeWNsZSBhbmQgdGhlIGRvd25saW5rIHJhdGUsIHRoZSB1cGdyYWRlIG9mIHRoZSBvbi1ib2FyZApkYXRhIGFjcXVpc2l0aW9uIHNvZnR3YXJlIHN0YWNrIGlzIGluIHByb2dyZXNzLiBUaGUgZ3JvdW5kIHNlZ21lbnQgaXMgYWxzbwpzdXBwb3J0ZWQgYnkgdGhlIHJhZGlvIGFtYXRldXIgY29tbXVuaXR5IGFuZCBpdCB0YWtlcyBhZHZhbnRhZ2Ugb2YgdGhlClNhdE5PR1MgbmV0d29yayBmb3IgaW5jcmVhc2VkIGRhdGEgZG93bmxpbmsgdm9sdW1lLgoKLS1fMDAwX1BIMFBSMDlNQjc4ODQwRDk2N0JGM0FBM0YxOUU2NEE5NkExREU5UEgwUFIwOU1CNzg4NG5hbXBfCg==',
```

Comment:
After base64 decoding, this becomes:

```eml
From: "Dutko, Dakota C"
        <dakota.c.dutko@nasa.gov>
To: "circulars@dev.gcn.nasa.gov" <circulars@dev.gcn.nasa.gov>
Subject: GRB 230207B: Detection by GRBAlpha
Thread-Topic: GRB 230207B: Detection by GRBAlpha
Thread-Index: Adk9dI5aYPr6M3WhQFmrIV1eoz5z1A==
Date: Fri, 10 Feb 2023 17:24:40 +0000
Accept-Language: en-US
Content-Language: en-US
Content-Type: multipart/alternative;
        boundary="_000_PH0PR09MB78840D967BF3AA3F19E64A96A1DE9PH0PR09MB7884namp_"
MIME-Version: 1.0

--_000_PH0PR09MB78840D967BF3AA3F19E64A96A1DE9PH0PR09MB7884namp_
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: quoted-printable


M. Dafcikova, J. Ripa (Masaryk U.), A. Pal (Konkoly Observatory), N. Werner
(Masaryk U.), M. Ohno, H. Takahashi (Hiroshima U.), L. Meszaros, B. Csak
(Konkoly Observatory), N. Husarikova, F. Munz , M. Topinka, M. Kolar, J.-P.
Breuer, F. Hroch (Masaryk U.), T. Urbanec, M. Kasal,  A. Povalac (Brno U.
of Technology), J. Hudec, J. Kapus, M. Frajt (Spacemanic s.r.o), R. Laszlo,
M. Koleda (Needronix s.r.o), M. Smelko, P. Hanak, P. Lipovsky (Technical U.
of Kosice), G. Galgoczi (Wigner Research Center/Eotvos U.), Y. Uchida, H.
Poon, H. Matake (Hiroshima U.), N. Uchida (ISAS/JAXA), T. Bozoki (Eotvos
U.), G. Dalya (Eotvos U.), T. Enoto (Kyoto U.), Zs. Frei (Eotvos U.), G.
Friss (Eotvos U.), Y. Fukazawa, K. Hirose (Hiroshima U.), S. Hisadomi
(Nagoya U.), Y. Ichinohe (Rikkyo U.), K. Kapas (Eotvos U.), L. L. Kiss
(Konkoly Observatory),  T. Mizuno (Hiroshima U.), K. Nakazawa (Nagoya U.),
H. Odaka (Univ of Tokyo), J. Takatsy (Eotvos U.), K. Torigoe (Hiroshima
U.), N. Kogiso, M. Yoneyama (Osaka Metropolitan U.), M. Moritaki (U.
Tokyo), T. Kano (U. Michigan) -- the GRBAlpha collaboration.

The long-duration GRB 230207B (AGILE/MCAL detection: GCN Circ. 33296;
Swift/BAT-GUANO detection: GCN Circ. 33298; CALET/GCBM detection: trigger
no. 1359779945; INTEGRAL/SPI-ACS detection: trigger no. 10190) was observed
by the GRBAlpha 1U CubeSat (Pal et al. Proc. SPIE 2020).

The detection was confirmed at the peak time 2023-02-07 04:40:48 UTC. The
T90 duration measured by GRBAlpha is 10 s and the overall significance
during T90 reaches 25 sigma.

The light curve obtained by GRBAlpha is available here:

https://gcc02.safelinks.protection.outlook.com/?url=3Dhttps%3A%2F%2Fgrbalph=
a.konkoly.hu%2Fstatic%2Fshare%2FGRB230207B_GCN.pdf&data=3D05%7C01%7Ccircula=
rs%40dev.gcn.nasa.gov%7C3fe277a638ff4981f1cb08db0b8bb301%7C7005d45845be48ae=
8140d43da96dd17b%7C0%7C0%7C638116467528236032%7CUnknown%7CTWFpbGZsb3d8eyJWI=
joiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%=
7C&sdata=3DeTEe3U86vBbOcaW8c0bwMhOV09PuLmhL2vYLLk3x7gY%3D&reserved=3D0

All GRBAlpha detections are listed at:
https://gcc02.safelinks.protection.outlook.com/?url=3Dhttps%3A%2F%2Fmonocer=
os.physics.muni.cz%2Fhea%2FGRBAlpha%2F&data=3D05%7C01%7Ccirculars%40dev.gcn=
.nasa.gov%7C3fe277a638ff4981f1cb08db0b8bb301%7C7005d45845be48ae8140d43da96d=
d17b%7C0%7C0%7C638116467528236032%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwM=
DAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3Dw=
b%2FKlmeBA%2FsvT7QCp6Cehjauoif4UEoJ4%2FMlc8Ln9k0%3D&reserved=3D0

GRBAlpha, launched on 2021 March 22, is a demonstration mission for a
future CubeSat constellation (Werner et al. Proc. SPIE 2018). The detector
of GRBAlpha consists of a 75 x 75 x 5 mm3 CsI scintillator read out by a
SiPM array, covering the energy range from ~50 keV to ~1000 keV. To
increase the duty cycle and the downlink rate, the upgrade of the on-board
data acquisition software stack is in progress. The ground segment is also
supported by the radio amateur community and it takes advantage of the
SatNOGS network for increased data downlink volume.

--_000_PH0PR09MB78840D967BF3AA3F19E64A96A1DE9PH0PR09MB7884namp_
```

Please:
1. [ ] Anonymize From address; replace with an email address that is at an [IANA example domain](https://www.iana.org/domains/reserved) like `example.com`.
2. [ ] Replace body with dummy text that is not derived from a real, published GCN Circular.
3. [ ] Make sure that the dummy text includes a liberal sprinkling of accented characters and unusual Unicode. Feel free to get really weird and include some RTL text, diacritics, ancient Sumerian glyphs, whatever.

Reply:
https://github.com/nasa-gcn/gcn.nasa.gov/issues/595